components/project/sections/NftInfo.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <n-card class="rounded-md text-start mb-3">
+    <div class="grid grid-cols-3 gap-3 align-top">
+      <div class="col-span-1">
+        <n-p>NFT image will be here</n-p>
+        <n-p class="text-xl">Description</n-p>
+        <n-p>NFT description will be here</n-p>
+      </div>
+      <div class="col-span-2">
+        <n-p class="text-4xl">{{ route }}</n-p>
+        <n-p>Token ID:</n-p>
+        <n-p>Token standard:</n-p>
+        <hr class="mb-6" />
+        <n-card class="rounded-md" title="Tags">
+          <n-p>Tag chips will be here</n-p>
+        </n-card>
+      </div>
+    </div>
+  </n-card>
+  <n-card class="rounded-md mb-3" title="Properties">
+    <n-data-table :columns="columns1" :data="data" :pagination="pagination" :bordered="false"
+  /></n-card>
```

Comment:
Please try not to keep tag endings on the new line. I know sometimes the formatting glitches it like this, but you can put them on the same line and that should fix it